src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -140,9 +140,9 @@ public HomeWorkspaceModel HomeSpace
             }
         }
 
-        public WorkspaceViewModel HomeSpaceViewModel
+        public HomeWorkspaceViewModel HomeSpaceViewModel
```

Comment:
This will be a API when you modify the return type.

Reply:
No worries will revert back.

Reply:
So according to @pinzart90 just changing return type to a more derived type might actually be safe.

Reply:
Apologies for the calamity! The change was done to allow the xaml file to resolve some of the bindings and make life easier. But it's not at all vital for the correct behavior of this bit of work, so I will revert it back to the higher cast level.